### PR TITLE
Fix wsClient and warning for /GET when listing projects.

### PR DESCRIPTION
**Pull request by mariadroman**
Fix wsClient:
- Create a wsClient in AppLoader and pass it to the required services.
- Close wsClient when the application closes.
- Avoid closing ws during the execution.

Fix /GET in /projects
- Add listPost/listGet methods to routes to differentiate access to the page from POST/GET
- Propagate changes accordingly.

### CODE REVIEW

File: app/AppLoader.scala [hunk 1]
```diff
@@ -39,11 +39,12 @@ class AppComponent(context: Context)(implicit val ec: ExecutionContext) extends
   with EhCacheComponents with I18nComponents {
 
   val config: Config = context.initialConfiguration.underlying
+  val wsClient = AhcWSClient()
 
   lazy val oauthGitHubService = new ApiOAuthGitHubService(config, AhcWSClient())
```
Comment (mahyaa) [inline app/AppLoader.scala]: Please change this one also to  wsClient.

File: app/services/impl/ApiTogglService.scala [hunk 2]
```diff
@@ -27,7 +27,6 @@ class ApiTogglService(ws: WSClient)(implicit val ec: ExecutionContext) extends T
         .withAuth(apiToken, "api_token", BASIC).get().map { response =>
         val bodyJSValue: JsValue = Json.parse(response.body)
         val validateBody = bodyJSValue.validate[List[Project]]
-        ws.close()
```
Comment (mahyaa) [inline app/services/impl/ApiTogglService.scala]: Where are you closing the ws?

File: app/AppLoader.scala (hunk 1 above)
Comment (mariadroman) [reply app/AppLoader.scala]: Nice catch! :)

File: app/services/impl/ApiTogglService.scala (hunk 2 above)
Comment (mariadroman) [reply app/services/impl/ApiTogglService.scala]: ws is being closed now on the AppLoader, when the application stops.

**Pull request merged**